src/main/groovy/de/dkfz/roddy/execution/jobs/SubmissionCommand.groovy

```diff
@@ -35,11 +62,10 @@ abstract class SubmissionCommand extends Command {
         String groupList = parentJobManager.getUserGroup()
         String accountName = job.customUserAccount ?: parentJobManager.getUserAccount()
         boolean holdJobsOnStart = parentJobManager.isHoldJobsEnabled()
-        boolean forcePassEnv = parentJobManager.enforcePassEnvironmentExportParameter
 
         // collect parameters for job submission
         List<String> parameters = []
-        if (forcePassEnv) parameters << getEnvironmentExportParameter()
+        parameters << " " + getEnvironmentExportParameter() + " "
```

Comment:
You shoudn't add spaces here, `parameters` is a list which is joined with spaces automatically